src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -319,13 +333,22 @@ def Configure(
                 metadata = ((md.key, md.value)
                             for md in request.metadata
                             if md.type == method_enum)
+                # For backward compatibility, do not change timeout when we
+                # receive a default value timeout.
+                if request.timeout_secs == 0:
+                    timeout_sec = channel_config.rpc_timeout_sec
```

Comment:
I don't think you forwarded `channel_config` to this method.

Reply:
Good catch. Since I can't do manual runs yet, this is one of those things that would have been difficult to find before running in staging.